web/src/user_events.ts

```diff
@@ -119,6 +119,7 @@ export const update_person = function update(event: UserUpdate): void {
             settings_org.maybe_disable_widgets();
             settings_org.enable_or_disable_group_permission_settings();
             settings.update_lock_icon_in_sidebar();
+            settings_account.remove_owner_from_role_select();
```

Comment:
I think it would be good to add code to live-update the disabled dropdown especially when a current owner makes another user owner so that they can change their role themselves.

Reply:
This will remove owner option from dropdown even if user is made an owner.

Reply:
This hasn't been fixed.
> This will remove owner option from dropdown even if user is made an owner.

And also the dropdown is not live-updated to show the updated role.